lib/app/modules/settings/views/theme_value_tile.dart

```diff
@@ -25,43 +26,115 @@ class ThemeValueTile extends StatefulWidget {
 }
 
 class _ThemeValueTileState extends State<ThemeValueTile> {
+    final _secureStorageProvider = SecureStorageProvider();
+    String appthem = '';
```

Comment:
this seems like a typo, it should probably be renamed to `appTheme` here, and everywhere else.

Reply:
Not actually. AppTheme is already used for light and dark. I can't save anything to it. It comes from system. I'm saving whatever I'm getting from SecureStorage to a string called appthem.

Reply:
I missed out on an e. Thanks for pointing it out.